databricks/koalas/series.py

```diff
@@ -5231,80 +5231,37 @@ def argsort(self) -> "Series":
         10    10
         dtype: int64
         """
-        internal = self.to_frame()._internal
-        index_scols = internal.index_spark_columns
-        data_scol = internal.data_spark_columns[0]
-        column_name = internal.column_labels[0][0]
-        spark_frame = internal.spark_frame.select(index_scols + [data_scol.alias(column_name)])
-
-        sdf = spark_frame.select(index_scols + [scol_for(spark_frame, column_name)])
-        cond = F.isnull(scol_for(sdf, column_name))
-        null_frame = sdf.filter(cond).select(index_scols + [F.when(cond, -1).alias(column_name)])
-        not_null_frame = None
-
-        sdf = sdf.dropna()
-        if len(null_frame.head(1)) != 0:
-            not_null_frame = sdf
-
-        seq_col_name = verify_temp_column_name(sdf, "__distributed_sequence_column__")
-        sdf = InternalFrame.attach_distributed_sequence_column(sdf, column_name=seq_col_name)
-        # sdf:
-        # +-------------------------------+-----------------+---+
-        # |__distributed_sequence_column__|__index_level_0__|  0|
-        # +-------------------------------+-----------------+---+
-        # |                              0|                0|  3|
-        # |                              1|                1|  3|
-        # |                              2|                2|  4|
-        # |                              3|                3|  1|
-        # |                              4|                4|  6|
-        # +-------------------------------+-----------------+---+
```

Comment:
Although I removed these comments while I was modifying, actually I like these. Could you add the same things with a new code in the original PR?

Reply:
Oh, yeah. I'll add it